### PR TITLE
redux-logger.d.ts updated to 2.0.0

### PR DESCRIPTION
**Pull request by arusakov**
@theaqua review please

### CODE REVIEW

File: redux-logger/redux-logger.d.ts [hunk 1]
```diff
@@ -8,13 +8,15 @@
 declare module 'redux-logger' {
 
   interface ReduxLoggerOptions {
+    actionTransformer?: (action: any) => any;
```
Comment (theaqua) [inline redux-logger/redux-logger.d.ts]: (action: object) => object

File: redux-logger/redux-logger.d.ts [hunk 2]
```diff
@@ -8,13 +8,15 @@
...
     collapsed?: boolean;
+    duration?: boolean;
     level?: string;
     logger?: any;
+    predicate?: (getState: Function, action: any) => boolean;
```
Comment (theaqua) [inline redux-logger/redux-logger.d.ts]: action: object

File: redux-logger/redux-logger.d.ts [hunk 3]
```diff
@@ -8,13 +8,15 @@
...
     timestamp?: boolean;
-    transformer?: (state:any)=>any;
-    predicate?: (getState:Function, action:any)=>boolean;
+    transformer?: (state:any) => any;
```
Comment (theaqua) [inline redux-logger/redux-logger.d.ts]: Redux's actions [must be plain objects](https://github.com/rackt/redux/blob/master/src/createStore.js#L95-L100), but I'm not sure about state. Maybe @gaearon can help?

File: redux-logger/redux-logger.d.ts (hunk 1 above)
Comment (arusakov) [inline redux-logger/redux-logger.d.ts]: Type "Object" in TS can not have properties besides defaults (toString, hasOwnProperty, etc). So in TypeScript-way action has type "any".

File: redux-logger/redux-logger.d.ts (hunk 2 above)
Comment (arusakov) [inline redux-logger/redux-logger.d.ts]: See above.

File: redux-logger/redux-logger.d.ts (hunk 3 above)
Comment (arusakov) [inline redux-logger/redux-logger.d.ts]: As actions state must be type "any", because the state can contain any of the properties.

**Comment by theaqua** [comment]
Ah. I think everything is ok.

**Comment by vvakame** [comment]
thanks all!

**Pull request merged**